src/components/text-style-button.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export interface textButtonInterface {
+    text: string
+    onClicked: () => void
+}
+
+
+const ButtonStyle = styled.div`
+    color: #BDBDBD;
+    font-family: Montserrat, Arial;
+    font-size: 14px;
+    cursor: pointer;
+    &:hover {
+        text-decoration: underline;
+    }
+`
```

Comment:
Nice CSS!

I mentioned in the ticket an easier way to change the default styling but I think we would do that just using styled components so **no code changes are needed to meet that Jira comment requirement**. As mentioned in the comment, we should change the pixel value to `rem`/`em`